deploy/docker/cp-dts-log-checker/dts-log-checker.py

```diff
@@ -0,0 +1,187 @@
+import glob
+import os
+from datetime import datetime
+import re
+import codecs
+import copy
+from pipeline.api import PipelineAPI
+from pipeline.log.logger import LocalLogger, TaskLogger, RunLogger
+
+DTS_LAST_RECORD_FILE = 'last_record.txt'
+DTS_EMAIL_TEMPLATE = 'template.html'
+DTS_LOGS_FILES = 'dts*.*'
+
+dts_user = os.environ.get('CP_DTS_LOG_NOTIFICATION_USER')
+if not dts_user:
+    raise RuntimeError("CP_DTS_LOG_NOTIFICATION_USER is not defined!")
+dts_users_copy = os.getenv('CP_DTS_LOG_NOTIFICATION_USERS_COPY', None)
+dts_users_copy_list = [cc_user.strip() for cc_user in dts_users_copy.split(
+    ",")] if dts_users_copy else []
+dts_notify_subject = os.environ.get(
+    'CP_DTS_LOG_NOTIFICATION_SUBJECT', 'DTS logs files has errors')
+dts_path = os.getenv('CP_DTS_LOG_CHECKER_SYSTEM_DIR')
+if not dts_path or not os.path.isdir(dts_path):
+    raise RuntimeError("CP_DTS_LOG_CHECKER_SYSTEM_DIR is not defined!")
```

Comment:
since we check that this directory exists we also need tell here that `CP_DTS_LOG_CHECKER_SYSTEM_DIR is not defined or doesn't exist on the system!`